components/admin_console/permission_schemes_settings/permission_schemes_settings.jsx

```diff
@@ -0,0 +1,68 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+import React from 'react';
+import {FormattedMessage} from 'react-intl';
+import {Link} from 'react-router-dom';
+
+export default class PermissionSchemesSettings extends React.PureComponent {
+    renderTitle() {
+        return (
+            <FormattedMessage
+                id='admin.permissions.permissionSchemes'
+                defaultMessage='Permission Schemes'
+            />
+        );
+    }
+
+    render = () => {
+        return (
+            <div className='wrapper--fixed'>
+                <h3 className='admin-console-header'>
+                    {this.renderTitle()}
+                </h3>
+
+                <div className={'banner info'}>
+                    <div className='banner__content'>
+                        <span>
+                            <FormattedMessage
+                                id='admin.permissions.introBanner'
+                                defaultMessage='Permission Schemes set the default permissions for Team Admins, Channel Admins and everyone else. Learn more about permission schemes in our documentation.'
```

Comment:
@esethna "Learn more about permission schemes in our documentation." Do we have documentation we can link to?